bundles/org.openhab.binding.homematic/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -127,6 +127,21 @@
 				<default>2048</default>
 				<advanced>true</advanced>
 			</parameter>
+			<parameter name="useAuthentication" type="boolean">
+				<label>Use authentication to access the gateway</label>
+				<description>If set to true, username and passwort is required.</description>
+				<advanced>true</advanced>
+				<default>false</default>
+			</parameter>
+			<parameter name="userName" type="text">
+				<label>User name for accessing the gateway</label>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="password" type="text">
+				<label>Password for accessing the gateway</label>
```

Comment:
Labels should be: <25 chars, max 2-3 words and properly capitalized.
(after this is fixed, don't forget top run the mvn i18n plugin)

Reply:
Done